accounts/models.py

```diff
@@ -275,3 +299,26 @@ def save(self, *args, **kwargs):
         )
 
         super().save(*args, **kwargs)
+
+
+class UserProfileReferralHit(models.Model):
+    """
+    List all users and they referred accounts (who signed up to the site)
+    """
+
+    user_profile = models.ForeignKey(UserProfile, on_delete=models.CASCADE)
+    referral_hit = models.OneToOneField(ReferralHit, on_delete=models.CASCADE)
+    # payment status
+    NONE = 0
```

Comment:
@autn what is the difference between `NONE`, `OPENED`, and `REQUESTING` here?

Reply:
@YPCrumble User A sends the link to user B, once B signed up, we create a record with payment status as `NONE`. When B becomes a "valid referral" (activate for 90 days), the daily job changes status to `OPENED`. If B requests the payout, the record will become `REQUESTING` along with the other payouts.

Does that make sense?